instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/go.mod

```diff
@@ -6,6 +6,7 @@ replace go.opentelemetry.io/contrib => ../../../../../
 
 require (
 	github.com/aws/aws-sdk-go-v2 v1.12.0
+	github.com/aws/aws-sdk-go-v2/service/dynamodb v1.11.0
```

Comment:
Is there a reason for this to use `v1.11.0` when `v1.12.0` of the sdk-go-v2 module is used?